webapp/tests/karma/unit/services/rules-engine.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable quotes */
```

Comment:
I think the `rules` var should be changed to use backticks instead of concatenating strings then we won't need to disable anything.